docs/pages/release_notes.rst

```diff
@@ -10,6 +10,7 @@ Hazelnut update (v5.4.2)
 Release date: 2023-08-20
 
 - Fixed issue where NutsEmployeeCredentials needed to be explicitly trusted when issued by another node
+- Reduced number of pages transmitted on a full sync to enhance performance
```

Comment:
This should be under v5.4.3?

Reply:
too hastily done, added v5.4.3 release notes.